src/code/entities/04_fishinggame.asm

```diff
@@ -1,24 +1,53 @@
-Data_004_5F28::
-    db   $00, $F0, $78, $01, $00, $F8, $7A, $01, $00, $00, $70, $01, $00, $08, $72, $01
-
-Data_004_5F38::
-    db   $00, $F0, $7C, $01, $00, $F8, $7E, $01, $00, $00, $70, $01, $00, $08, $72, $01
-
-Data_004_5F48::
-    db   $00, $F0, $78, $01, $00, $F8, $7A, $01, $00, $00, $74, $01, $00, $08, $76, $01
+FishermanRodLowSpriteRect::                 ; $5F28
```

Comment:
I think these are 4 bytes per entry (x,y,tile,attr)

Reply:
oh right ; I updated the list format.